instrument_config/paper_beam_setup_gain.pro

```diff
@@ -1,4 +1,4 @@
-FUNCTION paper_beam_setup_gain,obs,antenna,file_path_fhd=file_path_fhd,$
+FUNCTION paper_beam_setup_gain,obs,antenna,file_path_fhd=file_path_fhd,jdate_use=jdate_use,$
```

Comment:
jdate_use isn't actually used in this file. To catch keywords that may be passed in but are not used, add _extra=extra as the final keyword.